vms/avm/vm.go

```diff
@@ -757,8 +760,12 @@ func (vm *VM) verifyTransfer(tx UnsignedTx, in *avax.TransferableInput, cred ver
 func (vm *VM) verifyOperation(tx UnsignedTx, op *Operation, cred verify.Verifiable) error {
 	opAssetID := op.AssetID()
 
-	utxos := []interface{}{}
-	for _, utxoID := range op.UTXOIDs {
+	numUTXOs := len(op.UTXOIDs)
+	if numUTXOs > maxUTXOsPerOp {
+		return fmt.Errorf("operation has %d UTXO IDs but max is %d", numUTXOs, maxUTXOsPerOp)
+	}
```

Comment:
It's probably reasonable to put a limit on the number of UTXOs per op... If we want to add such a limit, it should be done inside of the operation's `Verify` function though. I'm not sure it makes sense to include this limit in this PR though, this check isn't required here.